### DISCUSSION TITLE
Should the `column_name` attribute create a `Column` variant with the same name?

### DISCUSSION THREAD

**Discussion by rybertm** [SeaQL/sea-orm#1504] [Q&A] [answered]
I was creating a macro to generate the appropriate `Column::#col` variant and was surprised the `column_name` attribute only changes the sql counterpart, not the generated `Column` variant itself. 

Since this attribute basically means "This column has this name on the database", isn't it more appropriate if the `Column` enum reflect that as well?

**Comment by tyt2y3** [comment, marked as answer]
I think it's better for the struct field's name align with Column's enum variant.
There have to be a mismatch somewhere in the end, anyway.

By the way please do not use a rhetorical question, you can express your statement in form of "I think it is more appropriate ..."
